### PR TITLE
Added brightness adjustment node

### PR DESCRIPTION
**Pull request by diwei-tan**
This PR adds a new draw node that allows to adjust the brightness of images to improve the visualisations of draw features if needed. Note that this uses PIL, and thus adds pillow to the requirements.txt.

### CODE REVIEW

File: peekingduck/pipeline/nodes/draw/brightness.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Adjusts brightness of an incoming image
+"""
+
+
+from typing import Any, Dict
+from PIL import Image, ImageEnhance
```
Comment (leeping-ng) [inline peekingduck/pipeline/nodes/draw/brightness.py:20]: As discussed, it's observed that Pillow decreases FPS by ~10% and introduces a new package. Let's stick with OpenCV calculations for brightness adjustment

Comment (diwei-tan) [reply peekingduck/pipeline/nodes/draw/brightness.py]: Found a particular opencv image manipulation method for Alpha (contrast) and beta (brightness) of images that follows `result = alpha * image(x,y) + beta`. Implemented contrast and brightness using this new method. This is much faster and does not affect fps noticeably.

File: peekingduck/pipeline/nodes/draw/utils/image_processor.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                                   brightness: int) -> np.array:
+    """ Process image according to contrast or brightness setting
+
+    Args:
+        frame (np.array): image of current frame
+
+    Returns:
+        processed_frame (np.array): processed image
+    """
+    return cv2.convertScaleAbs(frame, alpha=contrast, beta=brightness)
```
Comment (leeping-ng) [inline peekingduck/pipeline/nodes/draw/utils/image_processor.py:35]: What about removing this util function and putting this cv2 method in the node directly? Doesn't seem necessary to have a separate util at this point in time.

File: peekingduck/pipeline/nodes/draw/brightness.py (hunk 1 above)
Comment (leeping-ng) [reply peekingduck/pipeline/nodes/draw/brightness.py]: Nice!

File: peekingduck/pipeline/nodes/draw/utils/image_processor.py (hunk 2 above)
Comment (diwei-tan) [reply peekingduck/pipeline/nodes/draw/utils/image_processor.py]: shifted into node itself

**Pull request merged**